app/views/checklist_tests/_checklist_measure.html.erb

```diff
@@ -72,13 +75,38 @@
                           <i class = 'fa fa-fw fa-times text-danger' aria-hidden="true"></i>
                         <% end %>
                         <%= criteria_field.text_field :code, hide_label: false %>
+                          <div id="lookupModal" class="modal fade" role="dialog">
+                            <div class = "modal-dialog" id ="lookupModal-dialog">
+                              <div class="modal-content" id="lookupModal-content">
+                                <div class= "modal-header" id= "lookupModal-header">
+                                  <button type="button" class="close" id="lookupModal-close" data-dismiss="modal">&times;</button>
+                                    <h4 class="modal-title" id="lookupModal-title"> List of codes</h4>
+                                 </div>
+                                    <div class="modal-body" id="lookupModal-body">
+                                      <input type="text" id="lookupFilter" onkeyup="lookupFunction()" placeholder="Filter codes">
+                                      <ul class='list-unstyled' id="lookup_codes">
+                                      <% valuessets.each do |vs| %>
+                                        <% lookup_codevalues(vs, product_test.bundle).each do |pair_string| %>
+                                          <li class="valueset-listitem" href="#"><a><%= pair_string %></a></li>
```

Comment:
Each pair is being rendered as a link, but clicking it doesn't do anything. If these are supposed to lead somewhere then you should put in the proper `href`, but if not, then they shouldn't be rendered as links.